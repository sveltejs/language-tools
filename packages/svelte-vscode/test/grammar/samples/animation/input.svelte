{#each list as item, index (item)}
	<li animate:flip>{item}</li>
{/each}

{#each list as item, index (item)}
	<li animate:flip={{ delay: 500 }}>{item}</li>
{/each}
