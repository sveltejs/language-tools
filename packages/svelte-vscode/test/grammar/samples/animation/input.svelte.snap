>{#each list as item, index (item)}
#^ source.svelte meta.special.each.svelte meta.special.start.svelte punctuation.definition.block.begin.svelte
# ^ source.svelte meta.special.each.svelte meta.special.start.svelte punctuation.definition.keyword.svelte
#  ^^^^ source.svelte meta.special.each.svelte meta.special.start.svelte keyword.control.svelte
#      ^ source.svelte meta.special.each.svelte meta.special.start.svelte
#       ^^^^ source.svelte meta.special.each.svelte meta.special.start.svelte meta.embedded.expression.svelte source.ts
#           ^ source.svelte meta.special.each.svelte meta.special.start.svelte
#            ^^ source.svelte meta.special.each.svelte meta.special.start.svelte keyword.control.as.svelte
#              ^^^^^ source.svelte meta.special.each.svelte meta.special.start.svelte meta.embedded.expression.svelte source.ts
#                   ^ source.svelte meta.special.each.svelte meta.special.start.svelte punctuation.separator.svelte
#                    ^^^^^^^ source.svelte meta.special.each.svelte meta.special.start.svelte meta.embedded.expression.svelte source.ts
#                           ^ source.svelte meta.special.each.svelte meta.special.start.svelte meta.brace.round.svelte
#                            ^^^^ source.svelte meta.special.each.svelte meta.special.start.svelte meta.embedded.expression.svelte source.ts
#                                ^ source.svelte meta.special.each.svelte meta.special.start.svelte meta.brace.round.svelte
#                                 ^ source.svelte meta.special.each.svelte meta.special.start.svelte punctuation.definition.block.end.svelte
>	<li animate:flip>{item}</li>
#^ source.svelte text.svelte
# ^ source.svelte meta.scope.tag.li.svelte meta.tag.start.svelte punctuation.definition.tag.begin.svelte
#  ^^ source.svelte meta.scope.tag.li.svelte meta.tag.start.svelte entity.name.tag.svelte
#    ^ source.svelte meta.scope.tag.li.svelte meta.tag.start.svelte
#     ^^^^^^^ source.svelte meta.scope.tag.li.svelte meta.tag.start.svelte meta.directive.animate.svelte keyword.other.animation.svelte
#            ^ source.svelte meta.scope.tag.li.svelte meta.tag.start.svelte meta.directive.animate.svelte punctuation.definition.keyword.svelte
#             ^^^^ source.svelte meta.scope.tag.li.svelte meta.tag.start.svelte meta.directive.animate.svelte variable.function.svelte
#                 ^ source.svelte meta.scope.tag.li.svelte meta.tag.start.svelte punctuation.definition.tag.end.svelte
#                  ^ source.svelte punctuation.section.embedded.begin.svelte
#                   ^^^^ source.svelte meta.embedded.expression.svelte source.ts
#                       ^ source.svelte punctuation.section.embedded.end.svelte
#                        ^^ source.svelte meta.scope.tag.li.svelte meta.tag.end.svelte punctuation.definition.tag.begin.svelte
#                          ^^ source.svelte meta.scope.tag.li.svelte meta.tag.end.svelte entity.name.tag.svelte
#                            ^ source.svelte meta.scope.tag.li.svelte meta.tag.end.svelte punctuation.definition.tag.end.svelte
>{/each}
#^ source.svelte meta.special.each.svelte meta.special.end.svelte punctuation.definition.block.begin.svelte
# ^ source.svelte meta.special.each.svelte meta.special.end.svelte punctuation.definition.keyword.svelte
#  ^^^^ source.svelte meta.special.each.svelte meta.special.end.svelte keyword.control.svelte
#      ^ source.svelte meta.special.each.svelte meta.special.end.svelte punctuation.definition.block.end.svelte
>
>{#each list as item, index (item)}
#^ source.svelte meta.special.each.svelte meta.special.start.svelte punctuation.definition.block.begin.svelte
# ^ source.svelte meta.special.each.svelte meta.special.start.svelte punctuation.definition.keyword.svelte
#  ^^^^ source.svelte meta.special.each.svelte meta.special.start.svelte keyword.control.svelte
#      ^ source.svelte meta.special.each.svelte meta.special.start.svelte
#       ^^^^ source.svelte meta.special.each.svelte meta.special.start.svelte meta.embedded.expression.svelte source.ts
#           ^ source.svelte meta.special.each.svelte meta.special.start.svelte
#            ^^ source.svelte meta.special.each.svelte meta.special.start.svelte keyword.control.as.svelte
#              ^^^^^ source.svelte meta.special.each.svelte meta.special.start.svelte meta.embedded.expression.svelte source.ts
#                   ^ source.svelte meta.special.each.svelte meta.special.start.svelte punctuation.separator.svelte
#                    ^^^^^^^ source.svelte meta.special.each.svelte meta.special.start.svelte meta.embedded.expression.svelte source.ts
#                           ^ source.svelte meta.special.each.svelte meta.special.start.svelte meta.brace.round.svelte
#                            ^^^^ source.svelte meta.special.each.svelte meta.special.start.svelte meta.embedded.expression.svelte source.ts
#                                ^ source.svelte meta.special.each.svelte meta.special.start.svelte meta.brace.round.svelte
#                                 ^ source.svelte meta.special.each.svelte meta.special.start.svelte punctuation.definition.block.end.svelte
>	<li animate:flip={{ delay: 500 }}>{item}</li>
#^ source.svelte text.svelte
# ^ source.svelte meta.scope.tag.li.svelte meta.tag.start.svelte punctuation.definition.tag.begin.svelte
#  ^^ source.svelte meta.scope.tag.li.svelte meta.tag.start.svelte entity.name.tag.svelte
#    ^ source.svelte meta.scope.tag.li.svelte meta.tag.start.svelte
#     ^^^^^^^ source.svelte meta.scope.tag.li.svelte meta.tag.start.svelte meta.directive.animate.svelte keyword.other.animation.svelte
#            ^ source.svelte meta.scope.tag.li.svelte meta.tag.start.svelte meta.directive.animate.svelte punctuation.definition.keyword.svelte
#             ^^^^ source.svelte meta.scope.tag.li.svelte meta.tag.start.svelte meta.directive.animate.svelte variable.function.svelte
#                 ^ source.svelte meta.scope.tag.li.svelte meta.tag.start.svelte meta.directive.animate.svelte punctuation.separator.key-value.svelte
#                  ^ source.svelte meta.scope.tag.li.svelte meta.tag.start.svelte meta.directive.animate.svelte punctuation.section.embedded.begin.svelte
#                   ^^^^^^^^^^^^^ source.svelte meta.scope.tag.li.svelte meta.tag.start.svelte meta.directive.animate.svelte meta.embedded.expression.svelte source.ts
#                                ^ source.svelte meta.scope.tag.li.svelte meta.tag.start.svelte meta.directive.animate.svelte punctuation.section.embedded.end.svelte
#                                 ^ source.svelte meta.scope.tag.li.svelte meta.tag.start.svelte
#                                  ^ source.svelte meta.scope.tag.li.svelte meta.tag.start.svelte punctuation.definition.tag.end.svelte
#                                   ^ source.svelte punctuation.section.embedded.begin.svelte
#                                    ^^^^ source.svelte meta.embedded.expression.svelte source.ts
#                                        ^ source.svelte punctuation.section.embedded.end.svelte
#                                         ^^ source.svelte meta.scope.tag.li.svelte meta.tag.end.svelte punctuation.definition.tag.begin.svelte
#                                           ^^ source.svelte meta.scope.tag.li.svelte meta.tag.end.svelte entity.name.tag.svelte
#                                             ^ source.svelte meta.scope.tag.li.svelte meta.tag.end.svelte punctuation.definition.tag.end.svelte
>{/each}
#^ source.svelte meta.special.each.svelte meta.special.end.svelte punctuation.definition.block.begin.svelte
# ^ source.svelte meta.special.each.svelte meta.special.end.svelte punctuation.definition.keyword.svelte
#  ^^^^ source.svelte meta.special.each.svelte meta.special.end.svelte keyword.control.svelte
#      ^ source.svelte meta.special.each.svelte meta.special.end.svelte punctuation.definition.block.end.svelte
>